index.html

```diff
@@ -2614,6 +2639,90 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double outputLatency
+          </dt>
+          <dd>
+            <p>
+              The estimation in seconds of audio output latency, i.e. the
+              interval between the time the UA requests the host system to play
+              a buffer and the time at which the first sample in the buffer is
+              actually processed by the audio output device. For audible
+              devices such as speakers or headphones that produce an acoustic
+              signal, this latter time refers to the time when a sample's sound
+              is produced.
+            </p>
+            <p>
+              The <a><code>outputLatency</code></a> attribute value depends on
+              the platform and the hardware audio output device, the attribute
+              value does not change for the context's lifetime as long as audio
+              output device remains the same. If the audio output device is
+              changed the <a><code>outputLatency</code></a> attribute value
+              will be updated accordingly.
+            </p>
+          </dd>
+          <dt>
+            readonly attribute AudioTimeStamp outputTimeStamp
+          </dt>
+          <dd>
+            <p>
+              Holds an <a><code>AudioTimeStamp</code></a> instance containing
+              two correlated context's audio stream position values: the
+              <a href=
+              "#widl-AudioTimeStamp-contextTime"><code>contextTime</code></a>
```

Comment:
> I think I'd also try to work in your comment:

>>context.currentTime represents the sample frame immediately following the last sample-frame in the block of audio most recently processed by the context's rendering graph.

This is actually a quote from the `currentTime` attribute description, maybe there is a way to add a reference to this particular paragraph in order not to duplicate it?

> Also, contextTime is, I think, to similar to currentTime. Maybe outputContextTime to kind of match outputTimeStamp?

If possible I'd keep it generic, so that `AudioTimeStamp` is not bound to the particular attribute and can be reused (for example for input audio stream synchronization)

Thanks for your comments!